Gems/AWSCore/Code/Source/ScriptCanvas/AWSScriptBehaviorsComponent.cpp

```diff
@@ -12,35 +12,17 @@
 
 namespace AWSCore
 {
-    AZStd::vector<AZStd::unique_ptr<AWSScriptBehaviorBase>> AWSScriptBehaviorsComponent::m_behaviors;
-    bool AWSScriptBehaviorsComponent::m_alreadyAddedBehaviors = false;
-
-    void AWSScriptBehaviorsComponent::AddBehaviors()
-    {
-        if (!m_alreadyAddedBehaviors)
-        {
-            // Add new script behaviors here
-            m_behaviors.push_back(AZStd::make_unique<AWSScriptBehaviorDynamoDB>());
-            m_behaviors.push_back(AZStd::make_unique<AWSScriptBehaviorLambda>());
-            m_behaviors.push_back(AZStd::make_unique<AWSScriptBehaviorS3>());
-            m_alreadyAddedBehaviors = true;
-        }
-    }
-
     void AWSScriptBehaviorsComponent::Reflect(AZ::ReflectContext* context)
     {
-        AddBehaviors();
+        AWSScriptBehaviorDynamoDB::Reflect(context);
```

Comment:
Please add:
* Comment that new script behaviours need to be added to list below as you are no longer dynamically adding them
* Docs story to update how developers add AWS Script Behaviours.

Reply:
no comments needed, as originally it is not dynamically added.
no doc needed, as it is not specific to AWS, it is general way to create script canvas node, developer should have basic scripting knowledge to do it.